tests/lmeval/configs/fp8_dynamic_per_token.yaml

```diff
@@ -0,0 +1,13 @@
+cadence: "weekly"
+model: meta-llama/Meta-Llama-3-8B-Instruct
+scheme: FP8_DYNAMIC
+lmeval:
+  model_args:
+    dtype: bfloat16
```

Comment:
why did we have to add this?

Reply:
@kylesayrs  suggested having it, I think it will load up the float32 weights in bfloat16 to be used during QDQ